src/tcp-client-test/tcp-client-test.csproj

```diff
@@ -4,7 +4,7 @@
         <RootNamespace>tcp_client_test</RootNamespace>
         <Nullable>enable</Nullable>
         <IsPackable>false</IsPackable>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
When I ran tests and examples (with `dotnet run`) I was getting the error because I am using net8.0. However, I can create a script using net8.0 with `using QuestDB` and it all works. It would seem library is fine.